subsys/bluetooth/controller/hci/hci.c

```diff
@@ -28,6 +28,9 @@
 #include "hal/ccm.h"
 #include "ll_sw/pdu.h"
 #include "ll_sw/lll.h"
+#if defined(CONFIG_BT_CTLR_CONN_PROP_META)
```

Comment:
Having same guard here and inside the header file seems to be redundant.

Reply:
> Having same guard here and inside the header file seems to be redundant.

OK, removing conditional #include